key_management/Cargo.toml

```diff
@@ -21,6 +21,7 @@ serde_json = "1.0.57"
 serde_cbor = "0.11.1"
 log = "0.4.8"
 sodiumoxide = "0.2.6"
+hex = "0.4.2"
```

Comment:
Have you checked to see if we're using base64 anywhere in this module? If so, you should probably remove it, and also, perhaps look at the features line below